src/NuGet.Core/NuGet.Build.Tasks/NuGet.targets

```diff
@@ -78,6 +78,7 @@ Copyright (c) .NET Foundation. All rights reserved.
       _RestoreSolutionFileUsed=true;
       SolutionDir=$(SolutionDir);
       SolutionName=$(SolutionName);
```

Comment:
Seems like you accidentally removed the SolutionPath and SolutionExt. 

The tests will fail now.